p4-16/psa/PSA.mdk

```diff
@@ -451,25 +451,23 @@ sequence numbers in packets, for example, use Registers instead
 
 Direct counters are counters associated with a particular P4 table,
 and are implemented by the extern `DirectCounter`.  There are also
-indirect counters, which are implemented by the extern `Counter` (the
-name 'indirect' does not imply that there is any indirection involved
-in the implementation, only that they are not direct counters).  The
-primary differences between direct counters and indirect counters are:
+indexed counters, which are implemented by the extern `Counter`.  The
+primary differences between direct counters and indexed counters are:
```

Comment:
Shall we consider renaming `Counter` to `IndexedCounter`? It is clear from your last sentence that counters need an attribute (direct or indexed) so it would be nice to make those part of the name. I realize that this might be too much of a change, however, I wanted to throw it out there for discussion.